pkg/clients/compute/aks_test.go

```diff
@@ -1,44 +0,0 @@
-/*
```

Comment:
Looks like we don't have any tests for the AKS client. This is not a regression given the current status, are you just thinking we defer adding them since we anticipate a `v1beta1` refactor soon?

Reply:
I am. I'm surprised total coverage went up with this PR - presumably because there's less AKS specific code now that we're using the managed reconciler.